packages/mergebot/src/execute-pr-actions.ts

```diff
@@ -158,7 +158,7 @@ function doRestCall(call: RestMutation): Promise<void> {
     const url = `https://api.github.com/repos/DefinitelyTyped/DefinitelyTyped/${call.op}`;
     const headers = {
         "accept": "application/vnd.github.v3+json",
-        "authorization": `token ${process.env.BOT_AUTH_TOKEN}`,
```

Comment:
I would not recommend changing any of the token names; we have these set in the function app. If anything I would rather us become consistent and just do `GITHUB_TOKEN` like most other tools.

Reply:
oh, right. I got confused between the variables available to workflows vs functions.